src/quo2/components/list_items/dapp/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns quo2.components.list-items.dapp.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [quo2.components.list-items.dapp.style :as style]
+            [quo2.theme :as quo.theme]))
+
+(defn- view-internal
+  [{:keys [dapp state action blur? on-press theme]}]
```

Comment:
Perhaps the state :pressed should just be an internal var as it's local to the component depending on when the button is pressed or not.

Reply:
Could be... I copied the implementation of account list card, which passed code review last week. [link](https://github.com/status-im/status-mobile/pull/16893/files#diff-0edde74496b9aea848aab751fb462c62cdb2a4fdde3be2703d12c151bba59a0eR12)

Reply:
yeah but that would mean we have to handle the state of pressed externally, I don't think we want that. it probably just went unnoticed in that component you linked. the quo2 button component does this ok imo 👍

Reply:
I wrote a doc on the approach we should be following - I should have written this up a bit sooner as we figured this out a few months back 
https://www.notion.so/handling-custom-colors-d4060bc87cfb4c268d19a77f0711d3a1

the functionality/ customization overlay might need to be adjusted to also handle opacity in the color 👍

Reply:
yep, done